src/taipy/core/_entity/_reload.py

```diff
@@ -71,7 +72,9 @@ def _do_set_entity(self, *args, **kwargs):
             ]
 
             if not self._is_in_context:
-                entity_manager._set(self)
+                entity = _Reloader()._reload(manager, self)
+                fct(entity, *args, **kwargs)
```

Comment:
fct is applied twice. We should remove the previous call and move it to the else block.

Reply:
it's actually applied to different entities, the 1st fct called on line 64 applied to self `fct(self, *args, **kwargs)`, the 2nd on line 76 applied to the new up-to-date entity `fct(entity, *args, **kwargs)`

Reply:
YEs but the first one is useless if we set the second one, right?

Reply:
hmmm I believe we still need the 1st one to set the underscore value of the current entity, but what you said make sense too since we're saving directly to the repo, whatever is stored in the memory is now obsolete?

Reply:
so I tried putting the 1st fct called to the else block, and some tests failed :D apparently we do use _ attributes in some places and it needs to be updated

Reply:
Can we have an overview of all the _attributes of an entity used outside the entity itself?
That should clarify the impact of such change.